ui/app/constants/common.js

```diff
@@ -6,3 +6,5 @@ export const PRIMARY = 'PRIMARY'
 export const SECONDARY = 'SECONDARY'
 
 export const WAIT_FOR_NOTIFIER = 2000;
+
+export const WAIT_FOR_CONFIRMATION_DEFAULT = 2000;
```

Comment:
Can you explain with a comment on the code what is the purpose of this variable?

Reply:
Ok, i'll add a comment here